roles/os_hardening/README.md

```diff
@@ -283,6 +283,141 @@ We know that this is the case on Raspberry Pi.
+  - Default: `devtmpfs`
+  - Description: Set mount source for /dev
+- `os_mnt_dev_options` 
+  - Default: `'rw,nosuid,noexec'`
+  - Description: Configure mount options for /dev
+- `os_mnt_dev_filesystem` 
+  - Default: `ext4`
+  - Description: Configure file system for fstab entry /dev
+- `os_mnt_dev_shm_dir_mode` 
+  - Default: `1777`
+  - Description: Set default perimissions for /dev/shm
+- `os_mnt_dev_shm_enabled` 
+  - Default: `true`
+  - Description: Set to false to ignore /dev/shm mountpoint
+- `os_mnt_dev_shm_src` 
+  - Default: `tmpfs`
+  - Description: Set mount source for /dev/shm
+- `os_mnt_dev_shm_options` 
+  - Default: `rw,nosuid,nodev,noexec`
+  - Description: Configure mount options for /dev/shm
+- `os_mnt_dev_shm_filesystem` 
+  - Default: `ext4`
+  - Description: Configure file system for fstab entry /dev/shm
+- `os_mnt_home_dir_mode` 
+  - Default: `0755`
+  - Description: Set default perimissions for /home
+- `os_mnt_home_enabled` 
+  - Default: `false`
+  - Description: Set to true to configure /home mountpoint
+- `os_mnt_home_src` 
+  - Default: `""`
+  - Description: Set mount source for /home
+- `os_mnt_home_options` 
+  - Default: `rw,nosuid,nodev`
+  - Description: Configure mount options for /home
+- `os_mnt_home_filesystem` 
+  - Default: `ext4`
+  - Description: Configure file system for fstab entry /home
+- `os_mnt_run_dir_mode` 
+  - Default: `0755`
+  - Description: Set default perimissions for /run
+- `os_mnt_run_enabled` 
+  - Default: `true`
+  - Description: Set to false to ignore /run mountpoint
+- `os_mnt_run_src` 
+  - Default: `tmpfs`
+  - Description: Set mount source for /run
+- `os_mnt_run_options` 
+  - Default: `rw,nosuid,nodev`
+  - Description: Configure mount options for /run
+- `os_mnt_run_filesystem` 
+  - Default: `ext4`
+  - Description: Configure file system for fstab entry /run
+- `os_mnt_var_dir_mode` 
+  - Default: `0755`
+  - Description: Set default perimissions for /var
+- `os_mnt_var_enabled` 
+  - Default: `false`
+  - Description: Set to true to configure /var mountpoint
+- `os_mnt_var_src` 
+  - Default: `""`
+  - Description: Set mount source for /var
+- `os_mnt_var_options` 
+  - Default: `rw,nosuid,nodev`
+  - Description: Configure mount options for /var
+- `os_mnt_var_filesystem` 
+  - Default: `ext4`
+  - Description: Configure file system for fstab entry /var
+- `os_mnt_var_log_dir_mode` 
+  - Default: `0755`
+  - Description: Set default perimissions for /var/log
+- `os_mnt_var_log_enabled` 
+  - Default: `false`
+  - Description: Set to true to configure /var/log mountpoint
+- `os_mnt_var_log_src` 
+  - Default: `""`
+  - Description: Set mount source for /var/log
+- `os_mnt_var_log_options` 
+  - Default: `rw,nosuid,nodev,noexec`
+  - Description: Configure mount options for /var/log
+- `os_mnt_var_log_filesystem` 
+  - Default: `ext4`
+  - Description: Configure file system for fstab entry /var/log
+- `os_mnt_var_log_audit_dir_mode` 
+  - Default: `1777`
```

Comment:
This does not seem correct...

Reply:
I do not understand

In `roles/os_hardening/tasks/minimize_access.yml` there is

```
...
- name: Harden permissions for /var/log/audit directory
  file:
    dest: /var/log/audit
    owner: 'root'
    group: 'root'
    mode: '{{ os_mnt_var_log_audit_dir_mode }}'
...
```
In `roles/os_hardening/defaults.yml` there is 
```
...
os_mnt_var_log_filesystem: "ext4"

os_mnt_var_log_audit_dir_mode: '1777'
os_mnt_var_log_audit_enabled: false
os_mnt_var_log_audit_src: ""
...
```

And in README.md there is
```
- `os_mnt_var_log_audit_dir_mode`
  - Default: `1777`
  - Description: Set default perimissions for /var/log/audit
```

What am I missing?